docs/learn/smart-contract-developers/create-lsp7-token.md

```diff
@@ -1,29 +1,22 @@
 ---
-sidebar_label: '🌄 Create LSP7 Token'
+sidebar_label: '🌄 Create an LSP7 Token'
 sidebar_position: 2
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Create LSP7 Token
+# Create an LSP7 Token
 
-:::caution Disclaimer
+:::tip
 
-This guide might contain outdated information and will be updated soon.
+Please check the [Getting Started](./getting-started.md) for the project setup.
 
 :::
 
 ## Create a custom LSP7 Token contract
 
-:::tip
-
-Please read the [Introduction](./getting-started.md) page to set up your project correctly.
-
-:::
-
-We will now create a custom [LSP7 Digital Asset contract](../../standards/tokens/LSP7-Digital-Asset.md). This contract will extend [`LSP7Mintable`](../../contracts/contracts/LSP7DigitalAsset/presets/LSP7Mintable.md) & [LSP7Burnable](../../contracts/contracts/LSP7DigitalAsset/extensions/LSP7Burnable.md) (to allow burning tokens). We will also pre-mint 20k tokens to the owner of the contract (the deployer).
-To do that, delete the `Lock.sol` contract in the `contracts/` folder, then create a new file named `MyCustomToken.sol` with the following content:
+You will create a custom [LSP7 Digital Asset contract](../../standards/tokens/LSP7-Digital-Asset.md) that extends [`LSP7Mintable`](../../contracts/contracts/LSP7DigitalAsset/presets/LSP7Mintable.md) & [LSP7Burnable](../../contracts/contracts/LSP7DigitalAsset/extensions/LSP7Burnable.md) (to allow burning tokens). You will also pre-mint 20k tokens to the owner of the contract (the deployer).
```

Comment:
```suggestion
This guide will teach you how to create a custom [LSP7 Digital Asset](../../standards/tokens/LSP7-Digital-Asset.md) and pre-mint a certain amount of tokens to a specific address. The smart contract will use the following standardization: 

- [`LSP7Mintable`](../../contracts/contracts/LSP7DigitalAsset/presets/LSP7Mintable.md): Allow creating new assets on the smart contract
- [`LSP7Burnable`](../../contracts/contracts/LSP7DigitalAsset/extensions/LSP7Burnable.md): Allow tokens from being removed from the supply. 

You can modify the `mint()` function to adjust the amount of initially minted tokens and their receiver as described in the [LSP7 Mintable Documentation](../../contracts/contracts/LSP8IdentifiableDigitalAsset/presets/LSP8Mintable.md#mint). 
```

I would split the sentence using bullet points so it is more apparent to the reader what a specific standardization is used for.

In the beginning, try to keep the description vague. I think explaining that he will/must mint 20k tokens to his address is unnecessary. These are just parameters that can be changed individually and discussed later down the road.